README.md

```diff
@@ -275,9 +275,44 @@ It will monitor primary, secondary and SEO URLs:
 Optimizely Content Cloud supports only primary URLs and Optimizely Commerce supports all three types of URLs.
 
 There are two scheduled jobs:
-- *[Geta NotFoundHandler] Index content URLs* - as mentioned before, this job indexes URLs of content. Usually, it is required to run this job only once. All new content is automatically indexed. But if for some reasons content publish events are not firing when creating new content (for example, during the import), then you should set this job to run frequently.
+- *[Geta NotFoundHandler] Index content URLs* - as mentioned before, this job indexes URLs of content. Usually, it is required to run this job only once. All new content is automatically indexed. But if for some reason content publish events are not firing when creating new content (for example, during the import), then you should set this job to run frequently.
  - *[Geta NotFoundHandler] Register content move redirects* - this job creates redirects based on registered moved content. Normally, this job is not required at all, but there might be situations when content move is registered but redirect creation is not completed. This could happen during deployments. In this case, you can manually run this job or schedule it to run time to time to fix such issues.
 
+# Scheduled jobs
+
+Scheduled jobs powered by Coravel are included in the package, allowing jobs to run at set intervals.
+
+**Important**
+Use this only if the package is not intended for an Optimizely site. Optimizely has built-in scheduled jobs mechanism.
+
+To enable scheduled jobs, you need to configure the following settings:
+```
+services.AddNotFoundHandler(o =>
+{
+    ...
+    o.ScheduledJobs = true;
```

Comment:
I see how this naming could become confusing...

I suggest to rewrite this section with the following concepts in mind:

* Scheduled job - process that runs in background
** Suggestions cleanup job - shipped with the package, contains process that cleans up suggestions table
* Scheduler - mechanism that triggers scheduled jobs in a recurrent manner
** InternalScheduler - default scheduler, included in the core package, a scheduler that uses Coravel (link to the package docs)
** OptimizelyScheduler - uses Optimizely to schedule job runs